### PR TITLE
 feat(css-prefix): box-sizing support

### PR DESCRIPTION
**Pull request by magsout**
## Summary

Part of #1423.

Adds `box-sizing` related prefixes.

## Test Plan

ci

### CODE REVIEW

File: internal/compiler/transforms/compile/css-handler/prefix/prefixes/box-sizing.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {
+	createPrefixVisitor,
+	prefixCSSProperty,
+} from "@internal/compiler/transforms/compile/css-handler/prefix/utils";
+
+// https://github.com/Fyrd/caniuse/blob/main/features-json/css3-boxsizing.json
+// https://github.com/postcss/autoprefixer/blob/main/data/prefixes.js#L209-L211
+export default ["border-box", "content-box"].map((value) =>
+	createPrefixVisitor({
+		name: `box-sizing/${value}`,
```
Comment (jer3m01) [inline internal/compiler/transforms/compile/css-handler/prefix/prefixes/box-sizing.ts:10]: Just put the property name, no need for the value.
```ts
name: "box-sizing",
```
(can also remove the map)

**Review by jer3m01 — CHANGES_REQUESTED** [review]
As stated in the review, values in the name are only required when prefixing values

File: internal/compiler/transforms/compile/css-handler/prefix/prefixes/box-sizing.ts (hunk 1 above)
Comment (magsout) [reply internal/compiler/transforms/compile/css-handler/prefix/prefixes/box-sizing.ts]: done

**Comment by jer3m01** [comment]
Great thank you!

**Pull request merged**